docs/samples/edr-api-overview/edr-api-overview.md

```diff
@@ -75,6 +75,37 @@ This endpoint will perform the contract negotiation, transfer process and EDR st
 }
 ```
 
+The EDR negotiation returns only the id of the negotiation process that has been started.
+
+The EDR negotiation rely on two steps, contract negotiation and transfer process, both of which are asynchronous.
+In order to plug-in and get notified in every state of the two state machines, callbacks can be configured while starting
+the EDR negotiation:
+
+```json
+{
+  ...
+  "callbackAddresses": [
+    {
+      "uri": "http://localhost:8080/hooks",
+      "events": [
+        "transfer.process.started"
+      ],
+      "transactional": false
+    }
+  ]
+}
+```
+
+In this case we are interested only when the transfer process transition to the `STARTED` state.
+
+Once the EDR has been negotiated with the provider, the EDR itself will be stored in the configured vault and the metadata
+associated to it in the configured datasource for future querying.
+
+Since `tractusx-edc` v0.5.1 the cached EDRs also come with a state machine that will manage the lifecycle of an EDR
+on the consumer side. That means that it will auto-renew itself when the expiration date is approaching by
+firing another transfer process request with the same parameters as the original one. Once renewed the old-one
+will transition to the state `EXPIRED` and it will be removed from the database and the vault according to the [configuration](../../../core/edr-core/README.md).
```

Comment:
```suggestion
will transition to the `EXPIRED` state and it will be removed from the database and the vault according to the [configuration](../../../core/edr-core/README.md).
```